_data/identity.yml

```diff
@@ -1,4 +1,15 @@
 websites:
+    - name: 1Password
+      url: https://
+      img: 1password.png
+      tfa: No
+      email_address: support@1password.com
+      facebook: 1Password
+      twitter: 1Password
+      exceptions:
+        text: "Although AgileBits, the company behind 1Password, feel that two factor authentication doesn't really fit 1Password, we disagree. Because of the way in which 1Password handles authentication and password data."
```

Comment:
"Because of the way in which 1Password handles authentication and password data." sounds a bit like a sentence fragment. It could be merged with the previous sentence, to form `"Although AgileBits, the company behind 1Password, feel that two factor authentication doesn't really fit 1Password, we disagree. Because of the way in which 1Password handles authentication and password data."` or it could be rephrased to be two sentences. For example, `"AgileBits, the company behind 1Password, feels that two factor authentication doesn't really fit into 1Password's offering. We disagree, because of the way in which 1Password handles authentication and password data."`